gwt-regexp-j2cl-tests/pom.xml

```diff
@@ -77,22 +57,12 @@
                 <artifactId>j2cl-maven-plugin</artifactId>
                 <version>${maven.j2cl.plugin}</version>
                 <executions>
+                    <!-- to run this from command line, use mvn j2cl:test@j2cl-test -->
                     <execution>
                         <id>j2cl-test</id>
                         <goals>
                             <goal>test</goal>
                         </goals>
-                        <configuration>
-                            <defines>
-                                <gwt.enableDebugId>true</gwt.enableDebugId>
-                                <gwt.cspCompatModeEnabled>true</gwt.cspCompatModeEnabled>
-                                <gwt.strictCspTestingEnabled>true</gwt.strictCspTestingEnabled>
-                            </defines>
-                            <compilationLevel>BUNDLE</compilationLevel>
```

Comment:
no need to explicitly set the compilation level - plus, for tests this is likely to change to something new soon. if you really want it to be BUNDLE then that's okay, we can put it back